.github/workflows/builds.yml

```diff
@@ -30,6 +30,8 @@ jobs:
           S3_ACCESS_KEY: ${{ secrets.S3_ACCESS_KEY }}
           S3_SECRET_KEY: ${{ secrets.S3_SECRET_KEY }}
           S3_REGION: ${{ secrets.S3_REGION }}
+          S3_USE_PROXY: ${{ secrets.S3_USE_PROXY }}
+          S3_PROXY_URL: ${{ secrets.S3_PROXY_URL }}
```

Comment:
Can't the S3_ENDPOINT_URL be used instead? I'd be easier to minimize the number of config parmeters

Reply:
Actually, even removing the S3_USE_PROXY, just passing the endpoint url. independently on whether it's the proxy or not